### ISSUE TITLE
[bug] got RuntimeException when open g4 file in IntelliJ IDEA 

### ISSUE THREAD

**Issue by zwillim** [antlr/intellij-plugin-v4#530]
software version: IntelliJ IDEA 2021.3.2 
plugin version: ANTLR v4 1.17
plugin url: https://plugins.jetbrains.com/plugin/7358-antlr-v4/versions/stable/144492

when opening my g4 file, I got a RuntimeException. When I try to test my rule, it is said that 'Parsing was aborted'.

The Exception is as follows.
```
java.lang.RuntimeException: set is empty
	at org.antlr.v4.runtime.misc.IntervalSet.getMaxElement(IntervalSet.java:421)
	at org.antlr.v4.runtime.atn.ATNSerializer.serialize(ATNSerializer.java:169)
	at org.antlr.v4.runtime.atn.ATNSerializer.getSerialized(ATNSerializer.java:601)
	at org.antlr.v4.runtime.atn.ATNSerializer.getSerializedAsChars(ATNSerializer.java:605)
	at org.antlr.v4.tool.Grammar.createLexerInterpreter(Grammar.java:1313)
	at org.antlr.intellij.plugin.parsing.ParsingUtils.parseText(ParsingUtils.java:206)
	at org.antlr.intellij.plugin.ANTLRv4PluginController.lambda$parseText$1(ANTLRv4PluginController.java:437)
	at com.intellij.openapi.progress.util.BackgroundTaskUtil.lambda$computeInBackgroundAndTryWait$6(BackgroundTaskUtil.java:170)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:188)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$12(CoreProgressManager.java:624)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:698)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:646)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:623)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:66)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:175)
	at com.intellij.openapi.progress.util.BackgroundTaskUtil.lambda$computeInBackgroundAndTryWait$7(BackgroundTaskUtil.java:169)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:263)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

I found that this is because there is one line like this in my g4 file:
`NAME_EXT:[A-Za-z0-9\-\_][A-Za-z0-9\-_\.]*;`

**Comment by zwillim** [comment]
to be more detailed, it looks like the `\.` caused the problem.

**Comment by zwillim** [comment]
I found that the line was wrong and should be changed to `NAME_EXT:[A-Za-z0-9\-_][A-Za-z0-9\-_\\.]*;`. And after the change, the exception disappears.

**Comment by KvanTTT** [comment]
It should be fixed in the next version. The exception should not be thrown.

**Comment by KvanTTT** [comment]
By ANTLR itself.

**Comment by bjansen** [comment]
Thanks. Is this in 4.10.1, or in a future version?

**Comment by KvanTTT** [comment]
4.10.1

**Comment by bjansen** [comment]
Fixed in version 1.18 of the plugin (I suppose by this commit in antlr: https://github.com/antlr/antlr4/commit/7f07af85374383d877080fd97056977521056697)
